dds/DCPS/TypeObject.h

```diff
@@ -721,36 +758,36 @@ const DCPS::Encoding& get_typeobject_encoding();
   typedef Sequence<AppliedAnnotationParameter> AppliedAnnotationParameterSeq;
 
   struct AppliedAnnotation {
-    TypeIdentifierPtr                   annotation_typeid;
+    TypeIdentifier                   annotation_typeid;
```

Comment:
There are multiple spaces between the type and field name here and several other struct definitions. I'm not sure if it is our coding convention to do that.

Reply:
You are correct that it's not our coding convention.  These came from the spec's IDL file and were just preserved that way when they made their way into the header (by @jrw972 I think).  We could decide to reformat now, I'd be OK with it either way.

Reply:
I'm also good with either way. I think it's not critical at this point.

Reply:
I'd like it to be fixed and am willing to fix it when I have the time.

Reply:
I started on it already...